internal/kms/kms.go

```diff
@@ -334,18 +334,18 @@ type EncryptionKMS interface {
 	DecryptDEK(volumeID, encyptedDEK string) (string, error)
 }
 
-// DEKStoreType describes what DEKStore needs to be configured when using a
+// dekStoreType describes what DEKStore needs to be configured when using a
 // particular KMS. A KMS might support different DEKStores depending on its
 // configuration.
-type DEKStoreType string
+type dekStoreType string
```

Comment:
this is part of the public API. I think it is better to export it, just like the consts  below.

Reply:
ok.. lets keep it exported. thanks